src/rgw/rgw_lc.cc

```diff
@@ -683,7 +683,7 @@ int RGWLC::process(int index, int max_lock_secs)
     cls_rgw_lc_obj_head head;
     ret = cls_rgw_lc_get_head(store->lc_pool_ctx, obj_names[index], head);
     if (ret < 0) {
-      dout(0) << "RGWLC::process() failed to get obj head " << obj_names[index] << ret << dendl;
+      dout(0) << "RGWLC::process() failed to get obj head " << obj_names[index] << ", ret=" << ret << dendl;
```

Comment:
since we're touching the logs anyway, let's convert all dout calls to ldout, we already have a ctx

Reply:
@theanalyst Ok, I will take this later. Would you tell me the difference between dout and ldout? Thanks.